### PR TITLE
Add confirm for removing a friend

### PR DESCRIPTION
**Pull request by Syfaro**
This adds a confirm before removing a friend, as mentioned in #9.

### CODE REVIEW

File: static/scripts.js [hunk 1]
```diff
@@ -1668,4 +1668,22 @@
...
+            return;
+        }
+
+        $('form[name="frienduser"]').on('submit', function (e) {
+            e.preventDefault();
+
+            var shouldUnfriend = confirm('Are you sure you wish to remove this friend?');
+
+            if (shouldUnfriend) {
+                this.submit();
```
Comment (charmander) [inline static/scripts.js]:
```javascript
if (!shouldUnfriend) {
    e.preventDefault();
}
```
?

**Comment by charmander** [comment]
Confirmation messages are bad, but there’s really no nice way to implement undo here, so…!

**Comment by Syfaro** [comment]
Hopefully not that many are removing friends anyway!

**Comment by charmander** [comment]
Looks good; thanks!

**Pull request merged**